test/specs/scenarios/document.js

```diff
@@ -263,35 +269,41 @@ export const documentScenarios = async (lang) => {
           confirm,
           'national_identity_card.jpg'
         )
+        // first upload attempt should return an error if image quality is detected
+        // FIXME: Image quality errors are only returned for cut-off images in these tests!
+        // We should be able to define the error type for each request
         uploadFileAndClickConfirmButton(
           documentUpload,
           confirm,
-          'identity_card_with_cut-off_glare.png'
+          'identity_card_with_cut-off.png'
         )
-        // Multiple image quality warnings, display by priority
-        confirm.verifyImageQualityWarning(copy, 'cut-off')
+        confirm.verifyImageQualityMessage(copy, 'cut-off', 'error')
+        confirm.clickRedoButton()
 
         // 1st retake
-        confirm.clickRedoButton()
-        uploadFileAndClickConfirmButton(
-          documentUpload,
-          confirm,
-          'identity_card_with_glare.jpg'
-        )
-        confirm.verifyImageQualityWarning(copy, 'glare')
+        it('should return an error on the second back doc upload attempt', async () => {
```

Comment:
Minor: IMO the description should say it's the "1st retake attempt". It's a bit confusing with the `// 1st retake` comment and a description that says it is `"...the second back doc upload attempt"`. Similarly for line 295 also.